### PR TITLE
Propagate URL to Thoth Search UI in deployments

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to https://github.com/thoth-station/user-api/issues/1721

### CODE REVIEW

File: user-api/overlays/moc-prod/deploymentconfig.yaml [hunk 1]
```diff
@@ -126,6 +126,11 @@ spec:
                 secretKeyRef:
                   name: ceph
                   key: secret-key
+            - name: THOTH_SEARCH_UI_URL
+              valueFrom:
+                secretKeyRef:
+                  name: thoth-search-ui-url
+                  key: "https://thoth-station.ninja/search/"
```
Comment (fridex) [inline user-api/overlays/moc-prod/deploymentconfig.yaml:133]: Is this the key we want to use for the secret?

Comment (mayaCostantini) [reply user-api/overlays/moc-prod/deploymentconfig.yaml:133]: Right thank you (copy-paste error), I will replace it with `configMapKeyRef` 👍🏻

**Pull request merged**